### PR TITLE
Fix keyboard input focus in Graph Editor

### PR DESCRIPTION
**Pull request by HudsonHN**
Changed IsWindowFocused flags for inputs mentioned in #1424 to prevent all panels from responding.

### CODE REVIEW

File: source/MaterialXGraphEditor/Graph.cpp [hunk 1]
```diff
@@ -4135,16 +4136,19 @@ void Graph::drawGraph(ImVec2 mousePos)
             ed::NavigateToContent(0.0f);
         }
```
Comment (eytienne) [inline source/MaterialXGraphEditor/Graph.cpp:4137]: from my local experiments, I believe this small "centering" block could move above so you can get rid of the `shouldProcessInput` variable and wrap everything under the same `if`

**Review by eytienne — CHANGES_REQUESTED** [review]

File: source/MaterialXGraphEditor/Graph.cpp (hunk 1 above)
Comment (HudsonHN) [reply source/MaterialXGraphEditor/Graph.cpp]: Good catch! Changes pushed.

**Comment by jstone-lucasfilm** [comment]
@HudsonHN It would be great if we can include this improvement in MaterialX 1.39, which is scheduled for release this summer.  Would you mind retargeting your pull request to the `dev_1.39` branch, so that we can consider your change in that context?

**Comment by jstone-lucasfilm** [comment]
@HudsonHN We've now merged development work on MaterialX 1.39 back to the main branch of MaterialX, in preparation for wrapping up development on the 1.39.0 release. When you have a chance, could you retarget this pull request back to the main branch as well?

**Comment by HudsonHN** [comment]
> @HudsonHN We've now merged development work on MaterialX 1.39 back to the main branch of MaterialX, in preparation for wrapping up development on the 1.39.0 release. When you have a chance, could you retarget this pull request back to the main branch as well?

Fumbled a bit but it's done 😄

**Review by jstone-lucasfilm — APPROVED** [review]
This looks good to me, thanks @HudsonHN!

**Pull request merged**